src/common/types.cpp

```diff
@@ -991,8 +991,17 @@ struct EnumTypeInfo : public ExtraTypeInfo {
 	explicit EnumTypeInfo(string enum_name_p, vector<string> values_insert_order_p)
 	    : ExtraTypeInfo(ExtraTypeInfoType::ENUM_TYPE_INFO), enum_name(move(enum_name_p)),
 	      values_insert_order(std::move(values_insert_order_p)) {
+		for (auto &value : values_insert_order) {
+			if (!hash) {
+				hash = duckdb::Hash(value.c_str());
+			} else {
+				hash_t value_hash = duckdb::Hash(value.c_str());
+				hash = CombineHash(hash, value_hash);
+			}
+		}
 	}
 	string enum_name;
+	hash_t hash;
```

Comment:
Doesn't this need to be initialized?

Reply:
Mark was not a fan of this idea, I've ended up removing the hash. :)